### PR TITLE
merkleId Identity submodule submission

### PR DESCRIPTION
**Pull request by angelamerkelprebid**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [X] Other

## Description of change
This adds the Merkle id system to prebid github project as a submodule to the userId module. It requires two Merkle supplied identifiers, partner key (ptk) and publisher id (pubid). These must be supplied by merkleinc.com before the endpoint will work. Please get in touch at https://merkury.merkleinc.com/contact

### CODE REVIEW

**Comment by harpere** [comment]
@angelamerkelprebid - Tests failed.

**Comment by angelamerkelprebid** [comment]
docs PR is https://github.com/prebid/prebid.github.io/pull/2193

**Review by smenzer — CHANGES_REQUESTED** [review]
missing eids.js entry and a mismatch in defining the id object

File: test/spec/modules/eids_spec.js [hunk 1]
```diff
@@ -55,6 +55,20 @@ describe('eids array generation for known sub-modules', function() {
     });
   });
 
+  it('merkleId', function() {
+    const userId = {
+      merkleId: {
+        eid: 'some-random-id-value'
```
[the remark this answers is not in the document]
Comment (angelamerkelprebid) [reply test/spec/modules/eids_spec.js]: Thank you very much @smenzer !!

[the remark this answers is not in the document]
Comment (angelamerkelprebid) [reply test/spec/modules/eids_spec.js]: Tests appear to have passed this time.

**Review by smenzer — APPROVED** [review]

**Comment by angelamerkelprebid** [comment]
Thanks for the review @smenzer ! @jlukas79 looking forward to your comments.

**Comment by angelamerkelprebid** [comment]
Thanks so much @jlukas79 and @smenzer ! Could one of you kindly merge?

**Pull request merged**